### PR TITLE
Make linux build pass

### PR DESCRIPTION
**Pull request by tonyyang-svail**
build flag: `-DWITH_GPU=ON -DWITH_TESTING=ON -DCUDA_ARCH_NAME=Auto`

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -20,6 +20,12 @@ endif(APPLE)
 CheckCompilerCXX11Flag()
 set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++11")
 
+find_package(OpenMP)
```
Comment (wangkuiyi) [inline src/CMakeLists.txt]: I don't get it -- why do we need to check OpenMP?  Is that the old part of PaddlePaddle is using it?

File: CMakeLists.txt [hunk 2]
```diff
@@ -1,4 +1,4 @@
-cmake_minimum_required(VERSION 3.10)
+cmake_minimum_required(VERSION 3.0)
```
Comment (wangkuiyi) [inline CMakeLists.txt:1]: Thanks for the change. I noticed it is necessary from Mac side too.

**Review by wangkuiyi — APPROVED** [review]

File: src/CMakeLists.txt (hunk 1 above)
Comment (tonyyang-svail) [reply src/CMakeLists.txt]: I am getting this [error](https://github.com/flame/blis/issues/26) when test_tape is linking operators. Looks like some Ops is using OpenMP.

Comment (tonyyang-svail) [reply src/CMakeLists.txt]: @wangkuiyi  Like here https://github.com/PaddlePaddle/Paddle/blob/d00a0436b1c562060b49aa2981be094d78bcbdf5/paddle/fluid/operators/top_k_op.h#L58-L60

Comment (wangkuiyi) [reply src/CMakeLists.txt]: I see. How about we add a comment here?

Comment (tonyyang-svail) [reply src/CMakeLists.txt]: sure.

**Pull request merged**